network/ios/ios_facts.py

```diff
@@ -118,9 +118,14 @@
 import re
 
 from ansible.module_utils.basic import get_exception
-from ansible.module_utils.netcmd import CommandRunner
+from ansible.module_utils.netcli import CommandRunner, AddCommandError
 from ansible.module_utils.ios import NetworkModule
 
+def add_command(runner, command):
+    try:
+        runner.add_command(command)
+    except AddCommandError:
+        pass
```

Comment:
It might be worth adding a comment to here for future coders to say why this is OK and we return `pass`.